domain/ide/command/get_feature_flag_status.go

```diff
@@ -35,15 +38,25 @@ func (cmd *featureFlagStatus) Command() snyk.CommandData {
 
 func (cmd *featureFlagStatus) Execute(ctx context.Context) (any, error) {
 	if config.CurrentConfig().Token() == "" {
-		return nil, nil
+		return nil, errors.New("missing Snyk token")
```

Comment:
suggestion: should we maybe just say `not authenticated, cannot retrieve feature flags` instead of going down into the detail that it's a token missing?